radixdlt-core/radixdlt/src/integration/java/com/radixdlt/integration/distributed/simulation/tests/consensus_ledger_sync/FallBehindMultipleEpochsLedgerSyncTest.java

```diff
@@ -0,0 +1,170 @@
+ */
+
+package com.radixdlt.integration.distributed.simulation.tests.consensus_ledger_sync;
+
+import com.google.inject.AbstractModule;
+import com.google.inject.Provides;
+import com.google.inject.TypeLiteral;
+import com.radixdlt.SyncModuleRunner;
+import com.radixdlt.consensus.bft.BFTNode;
+import com.radixdlt.consensus.bft.BFTValidatorSet;
+import com.radixdlt.consensus.bft.Self;
+import com.radixdlt.consensus.bft.View;
+import com.radixdlt.counters.SystemCounters.CounterType;
+import com.radixdlt.environment.EventProcessor;
+import com.radixdlt.environment.ProcessWithSyncRunner;
+import com.radixdlt.environment.RemoteEventProcessor;
+import com.radixdlt.environment.ScheduledEventDispatcher;
+import com.radixdlt.environment.rx.ModuleRunnerImpl;
+import com.radixdlt.environment.rx.RemoteEvent;
+import com.radixdlt.epochs.EpochsLedgerUpdate;
+import com.radixdlt.integration.distributed.simulation.NetworkLatencies;
+import com.radixdlt.integration.distributed.simulation.NetworkOrdering;
+import com.radixdlt.integration.distributed.simulation.SimulationTest;
+import com.radixdlt.integration.distributed.simulation.DelayedSyncModuleRunner;
+import com.radixdlt.integration.distributed.simulation.ConsensusMonitors;
+import com.radixdlt.integration.distributed.simulation.LedgerMonitors;
+import com.radixdlt.integration.distributed.simulation.SimulationTest.Builder;
+import com.radixdlt.sync.SyncConfig;
+import com.radixdlt.sync.messages.local.LocalSyncRequest;
+import com.radixdlt.sync.messages.local.SyncCheckReceiveStatusTimeout;
+import com.radixdlt.sync.messages.local.SyncCheckTrigger;
+import com.radixdlt.sync.messages.local.SyncRequestTimeout;
+import com.radixdlt.sync.messages.remote.StatusRequest;
+import com.radixdlt.sync.messages.remote.StatusResponse;
+import com.radixdlt.sync.messages.remote.SyncRequest;
+import com.radixdlt.sync.messages.remote.SyncResponse;
+import io.reactivex.rxjava3.core.Flowable;
+import io.reactivex.rxjava3.core.Observable;
+import org.junit.Test;
+
+import java.time.Duration;
+import java.util.Set;
+import java.util.concurrent.TimeUnit;
+import java.util.function.Function;
+import java.util.stream.IntStream;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * A node that falls behind multiple epochs should be able to successfully sync its ledger
+ * and keep up once synced.
+ */
+public class FallBehindMultipleEpochsLedgerSyncTest {
+	private static final int NODE_UNDER_TEST_INDEX = 2;
+	private static final long SYNC_DELAY = 5000L;
+
+	private final Builder testBuilder;
+
+	public FallBehindMultipleEpochsLedgerSyncTest() {
+		this.testBuilder = SimulationTest.builder()
+			.numNodes(3)
+			.networkModules(
+				NetworkOrdering.inOrder(),
+				NetworkLatencies.fixed(10)
+			)
+			.addSingleByzantineModule(NODE_UNDER_TEST_INDEX, new AbstractModule() {
```

Comment:
Rather than doing it this way, it would be better to just use the regular runner and modify `SimulationNodes.start()` specifying if any runners shouldn't be initialized. Also should just remove `SyncModuleRunner` as it's an unneeded hack.

Reply:
done